.github/workflows/test_notebooks.yml

```diff
@@ -5,15 +5,19 @@ on:
   push:
     branches: [ develop, stable, nbtests ]
     paths-ignore:
-      - '**/*.md' # ignore markdown files
-      - '**/*.rst' # ignore restructured text files
+      - 'DEA_Sandbox.ipynb' # ignore landing page
+      - 'DEA_notebooks_template.ipynb' # ignore template
+      - '*.md' # ignore all markdown files
+      - '*.rst' # ignore all restructured text files
```

Comment:
I think that this `**/*` syntax is needed to ignore all of them recursively, not just in the root directory.

```yaml
      - '**/*.md' # ignore markdown files
      - '**/*.rst' # ignore restructured text files
```

Reply:
I don't think the recursive ignore was ignoring the top level markdown files in your previous PR - it seemed to be only catching the ones nested within subfolders.

I'd be happy with leaving the recursive ignore as it is though and adding in your suggested top level "*" ignore to catch the Readme?

Reply:
Actually, maybe it just needs to be "**.md?"
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-file-paths